spec/spec-suite-jasmine.js

```diff
@@ -0,0 +1,8 @@
+'use babel'
+/* @flow */
+
+require('./spec-suite')
+
+module.exports = function () {
```

Comment:
I guess for top-level functions, it's a little weird, because I know I would never want to bind `this` in there.